cmake/templates/cmake_toolchains.rst.in

```diff
@@ -14,13 +14,13 @@
 |cmake| toolchains shipped with |hpx|
 =====================================
 
-In order to compile HPX for various platforms, we provide a variety of toolchain
-files that take care of setting up various |cmake| variables like compilers etc.
+In order to compile |HPX| for various platforms, we provide a variety of toolchain
```

Comment:
```suggestion
In order to compile |hpx| for various platforms, we provide a variety of toolchain
```

Reply:
This needs to be lower case to render as italic *HPX*.

Reply:
I'll go back and change it--thanks!